### ISSUE TITLE
adding Additional Details

### ISSUE THREAD

**Issue by akeilox** [alimertozdemir/EPassportNFCReader#3]
I can get the basic details for the ID and PP, but cant get the additional data on the chip. These are becoming more common with ID or Passport chips where additional info is stored.

It would be great if you can add Additional Details like in the other example (also based on tananev jmrtd)

https://github.com/jllarraz/AndroidPassportReader/blob/master/app/src/main/java/example/jllarraz/com/passportreader/data/AdditionalPersonDetails.kt

**Comment by alimertozdemir** [comment]
Hi @akeilox ,
I have just added some additional data groups including signature, fingerprint and additional personal data. I hope it helps. 

https://github.com/alimertozdemir/EPassportNFCReader/blob/8914e6da64bc35e72900364436c1591728bcb15c/app/src/main/java/com/alimert/passportreader/ui/MainActivity.java#L371

**Comment by akeilox** [comment]
Hi @alimertozdemir thank you for the update. I have just tested and it appears to work well, definitely helps. I do believe fingerprint is further encoded with special key only immigration can read. I have tested additional personal data so far.

On related note, are there any plans on extending the tvResult View, like with HTML view? So the information can be better formatted, image resized, for better viewing or submitting to an intranet page ? Like hiding buttons and showing results as HTML.
I realised when I leave the phone on the passport and read/scroll through details it accidentally tried to re-read the passport.
Would be interested to hear your thoughts if you plan to improve upon it or leave it as it is.
